README.md

```diff
@@ -1 +1,27 @@
-# ToDoList_JS_App
\ No newline at end of file
+# ToDo List
+This project is build by javascript web packages which can add and remove daily tasks.
+
+# Built With
+Html
+Css
+Javascript
+Sublime Text
+
+# Author
+👤 Sahar Saba Amiri
+
+Gmail: saharsaba.amiri123@gmail.com
+LinkedIn: https://www.linkedin.com/sahar-saba-amiri
+GitHub : https://github.com/Sahar-SE
+
```

Comment:
- [ ] [Optional] Kindly write the step to `clone, install and run the app` on the browser So any user who wants to see your project can run it by looking at the steps and I would suggest you provide the `live demo` on your readme file So in the future recruiters can get access to your project easily which create positive impact on the job.